Ensembl/ensembl-client
# Integrate InstantDownloadTranscript in zmenu

## Type
- New feature
- Bugfix

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-556

## Description
This PR adds the InstantDownloadTranscript component to the Zmenu in GenomeBrowser

Testing scenario:
- visit Genome Browser page
- click on a feature inside the Genome Browser canvas
- in the zmenu that will pop up, click on "Download"
- observe the InstantDownload component

Note: non-protein-coding features should not show checkboxes related to CDS or protein sequence.

**Also in this PR:**
- added `useApiService` hook to use the api service from inside a component (in the future, might consider switching to [react-query](https://github.com/tannerlinsley/react-query))
- fixed a bug in InstantDownloadTranscript component (it was not hiding protein-related checkboxes for non-protein-coding features)
- updated styling (including that of the Loader) after consultation with Andrea

## Views affected
Genome Browser page

## Feature deployed to
http://hx-rke-wp-webadmin-14-worker-1.caas.ebi.ac.uk:31372
